contracts/TokenManager.sol

```diff
@@ -260,4 +600,11 @@ contract TokenManager is System, IApplication {
     }
     return symbolMatch;
   }
+
+  function convertToBep2Amount(uint256 amount, uint256 bep20TokenDecimals) internal pure returns (uint256) {
+    if (bep20TokenDecimals > BEP2_TOKEN_DECIMALS) {
+      return amount.div(10**(bep20TokenDecimals-BEP2_TOKEN_DECIMALS));
```

Comment:
What will happen if `bep20TokenDecimals-BEP2_TOKEN_DECIMALS` is quite large that overflow? Can you have a test case for this?

Reply:
Good idea.